hack/build-go.sh

```diff
@@ -10,7 +10,11 @@ STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/common.sh"
 source "${OS_ROOT}/hack/util.sh"
-os::log::install_errexit
+source "${OS_ROOT}/hack/lib/util/trap.sh"
```

Comment:
We need to `source "${OS_ROOT}/hack/lib/log/stacktrace.sh` inside of `hack/lib/util/trap.sh` to call the stacktrace functions inside of the error handler. If we also `source` the other direction, we hot-loop and segfault. 

We *could* make a dummy-function like `os::lib::log::stacktrace` and make the `source` conditional if we see that function in what is currently `set` but that seems *a)* crazy and *b)* less ideal than just `source`-ing both from your script.

However, if we are attempting to slim down the preamble in our scripts I can see how it could be useful. We could even just have a `source "${OS_ROOT}/hack/lib.sh` that `source`s the entire lib tree in such a manner.

@smarterclayton thoughts?